### DISCUSSION TITLE
If we have 13 months of the year, how should we name them?

### DISCUSSION THREAD

**Discussion by juancProfero** [matizar/tupido#4] [General]
When making an alternative **calendar of 13 months**, we should think about what those months should be called, because what we know is that the names of the months that we have today do not make much sense because of the following: **October**, month 10: its name **comes from number 8**, November, month 11, his name comes from the number 9, July his name comes from the Roman politician Julius Caesar, August: his name comes from the grandson-nephew of Julius Caesar.

**Comment by matizar** [comment]
If we use the zodiacal signs, to name the months of the year we can use one of the 13 characters or signs to referring to each one.

Month 01: ♐ Sagittarius 
Month 02: ♑ Capricorn

A date can be: ⛎ 12 - 2022

**Comment by ghost** [comment]
I share a version of a calendar of 13 months programmed in 8 languages github.com/juanmanuelvelasquezalvarez/ideal
